getting-started/scaling-out.md

```diff
@@ -30,72 +29,65 @@ and removed using [`delete_data_node`][delete_data_node].
 
 Note that:
 
-* Data nodes are added as objects locally and connect to an
-  existing PostgreSQL instance.
+* A data node is added as a local object on the access node and
+  contains the configuration needed to connect to a database on a
+  PostgreSQL instance.
 
-* You should already have a running PostgreSQL server on the data node host.
+* You should already have a running PostgreSQL server on an instance
+  that will host a data node database. The data node's database will
+  be created when executing the `add_data_node` command on the access
+  node and should _not_ exist prior to adding the data node.
 
-* Ensure that the data node has password authentication enabled
-  in their `pg_hba.conf` files for any non-superusers.
+* PostgreSQL instances that will act as data nodes are assumed to
+  contain the same roles and permissions as the access node instance
+  (or, at a minimum, the roles and permissions associated with
+  distributed hypertables). Currently, such roles and permissions need
```

Comment:
Even though correct, I suggest that you remove the comment in the parentheses. It's a good way to distract the reader from the main focus.